spec/FilesController.spec.js

```diff
@@ -70,7 +70,10 @@ describe('FilesController', () => {
         expect(log1.level).toBe('error');
 
         const log2 = logs.find(
-          x => x.message === 'Could not store file: yolo.txt.'
+          // we can no longer expect 'Could not store file: yolo.txt' because we now
```

Comment:
nit: change message so it makes sense without prior knowledge of how it worked before this change